Created separate issue templates for enhancements/ fixes (#237)
astronomer/docs: .github/ISSUE_TEMPLATE/request-template.md

---
name: Request New Documentation
about: Requests for new docs, topics, or enhancements
title: ''
labels:
assignees: ''

---

**Request type:**

- [ ] **New document:**: Add the `newcontent` label to this issue. 
- [ ] **Significant change to an existing document**: Add the `enhancement` label to this issue. 

**What is the purpose of the document/enhancement? Select all that apply:**

> - [ ] Setup steps a new feature for an upcoming release (Add the appropriate version label to this issue). 

Reviewer: ```suggestion
- [ ] Document a new feature for an upcoming release. Add the appropriate release version label to this issue. 
```

---
name: Request New Documentation
about: Requests for new docs, topics, or enhancements
title: ''
labels:
assignees: ''

---

> **What kind of documentation change are you requesting?**

Reviewer: ```suggestion
**Kind**
```

* **New document, topic, or other piece of content**: Add the `newcontent` label to this issue. 
* **Significant change to an existing document**: Add the `enhancement` label to this issue. 

> * **Fix to an existing document**: See below for what labels to add

Reviewer: ```suggestion
* **Fix to an existing document**: See below for what labels to add
```

Should this not be here since we have the fix template?

  
**For new documents and enhancements, answer the following questions**

_What is the purpose of the document/enhancement? Select all that apply:_
- [ ] Setup steps a new feature for an upcoming release (Add the appropriate version label to this issue). 
- [ ] Information for an existing feature that isn't yet documented.
- [ ] Additional considerations/ reference material for an existing feature or document.

> **Describe the problem/ feature that requires documentation:**

Reviewer: ```suggestion
**Problem/feature that requires documentation:**
```

> **What key information should be included in the document?**

Reviewer: ```suggestion
**Information to be added/included**
```
